HAB/HAB.ino

```diff
@@ -49,7 +49,9 @@ void loop() {
     Sensors::PressureData pressure = Sensors::getPressureData();
 
     Logging::MissionData _data;
-    _data.setGpsData(gpsData);
+    if(gpsData.isValid) {
+      _data.setGpsData(gpsData);
```

Comment:
Let's move this line into the existing `if (gpsData.isValid)` statement at line 64.

Reply:
Done, please double check if I did it correctly! Thank you!